src/cli/peakdetector/PeakDetectorCLI.cpp

```diff
@@ -556,11 +556,13 @@ void PeakDetectorCLI::loadCompoundsFile() {
 		int loadCount = DB.loadCompoundCSVFile(mavenParameters->ligandDbFilename);
 		mavenParameters->compounds = DB.compoundsDB;
 		if (loadCount == 0) {
-			cerr << "Warning: Given compound database is empty!" << endl;
-		} else {
+			cerr << "Warning: Uploaded compound database is empty!" << endl;
```

Comment:
I hope this change was not intentional and probably a rebase thing, because the changes seen here are that of a previous commit over which changes were made and merged to develop as [suggested](https://github.com/ElucidataInc/ElMaven/pull/863#discussion_r217343033).